TemplateHandler.csproj

```
@@ -25,5 +25,17 @@
       Exclude="Template.ProcessCube.Project\**\bin\**;Template.ProcessCube.Project\**\obj\**;Template.ProcessCube.Project\**\node_modules\**"/>
     <Compile Remove="**\*" />
   </ItemGroup>
-
+  <Target Name="Template">
+    <Message Importance="High" Text="Create Template in $(LocalNuget):" />
+    <RemoveDir Directories="$(LocalNuget)" />
+    <Exec Command="dotnet pack -o $(LocalNuget)"  ConsoleToMSBuild="true" />
+  </Target>
+  <Target Name="Uninstall-Template">
+    <Exec Command="dotnet new uninstall Template.ProcessCube.Project" ConsoleToMSBuild="true" />
+  </Target>
+  <Target Name="Install-Template">
```

```suggestion
  <Target Name="Template">
    <Message Importance="High" Text="Create Template in $(LocalNuget):" />
    <RemoveDir Directories="$(LocalNuget)" />
    <Exec Command="dotnet pack -o $(LocalNuget)"  ConsoleToMSBuild="true" />
  </Target>
  <Target Name="UninstallTemplate">
    <Exec Command="dotnet new uninstall Template.ProcessCube.Project" ConsoleToMSBuild="true" />
  </Target>
  <Target Name="InstallTemplate">
```
Sonst hattest du ja auch keine Bindestriche benutzt